client/src/components/Collections/common/CollectionEditView.vue

```diff
@@ -24,23 +156,39 @@
                             @clicked-save="clickedSave" />
                     </div>
                 </DbKeyProvider>
-            </b-tab>
+            </BTab>
+
             <SuitableConvertersProvider :id="collectionId" v-slot="{ item }">
-                <b-tab
+                <BTab
                     v-if="item && item.length"
                     title-link-class="collection-edit-convert-datatype-nav"
-                    @click="updateInfoMessage(newCollectionMessage)">
-                    <template v-slot:title> <FontAwesomeIcon icon="cog" /> &nbsp; {{ l("Convert") }}</template>
+                    @click="updateInfoMessage('This will create a new collection in your History.')">
+                    <template v-slot:title>
+                        <FontAwesomeIcon :icon="faCog" />
+                        &nbsp; {{ localize("Convert") }}
+                    </template>
+
                     <SuitableConvertersTab :suitable-converters="item" @clicked-convert="clickedConvert" />
-                </b-tab>
+                </BTab>
             </SuitableConvertersProvider>
-            <b-tab
+
+            <BTab
                 v-if="isConfigLoaded && config.enable_celery_tasks"
                 title-link-class="collection-edit-change-datatype-nav"
-                @click="updateInfoMessage(expectWaitTimeMessage)">
-                <template v-slot:title> <FontAwesomeIcon icon="database" /> &nbsp; {{ l("Datatypes") }} </template>
+                @click="
+                    updateInfoMessage(
+                        'This operation might take a short while, depending on the size of your collection.'
+                    )
+                ">
+                <template v-slot:title>
+                    <FontAwesomeIcon :icon="faDatabase" />
+                    &nbsp; {{ localize("Datatypes") }}
+                </template>
+
                 <DatatypesProvider v-slot="{ item, loading }">
```

Comment:
Maybe it is a good time to drop this "old" provider and use one of the modern composables we have around Datatypes. This will simplify the type handling here. But it can be in a follow up if you want.

Reply:
There will be many changes, so I will create a follow-up PR to replace them.